src/runtime/pkg/containerd-shim-v2/shim_management.go

```diff
@@ -183,8 +183,13 @@ func (s *service) mountPprofHandle(m *http.ServeMux, ociSpec *specs.Spec) {
 	m.Handle("/debug/pprof/trace", http.HandlerFunc(pprof.Trace))
 }
 
-// SocketAddress returns the address of the abstract domain socket for communicating with the
+// GetSandboxesStoragePath returns the storage path where sandboxes info are stored
+func GetSanboxesStoragePath() string {
```

Comment:
It makes sense to split the `unix://` part from the path, but I don't think it makes that much sense to dynamically build the path. I would have been happy with returning `run/vc/sbs` here (the `unix://` makes it non-portable anyway, so there is little benefit from using `filePath.Separator`).

Not worth a respin.

Reply:
I kept what is in the SocketAddress() below, but you are right, with the unix:// prefix there is no more need for the filePath.Separator at all.

Reply:
nit:
```suggestion
func GetSandboxesStoragePath() string {
```